### PR TITLE
Prepare Mastodon for zeitwerk autoloader

### PR DESCRIPTION
**Pull request by ClearlyClaire**
Add inflections and rename/move a few classes.

In particular, `app/lib/exceptions.rb` and `app/lib/sanitize_config.rb` were manually loaded while still in autoload paths.

### CODE REVIEW

File: app/controllers/authorize_interactions_controller.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def set_resource
 
   def located_resource
     if uri_param_is_url?
-      ResolveURLService.new.call(uri_param)
+      ResolveUrlService.new.call(uri_param)
```
Comment (nightpool) [inline app/controllers/authorize_interactions_controller.rb:41]: `inflect.acronym 'URL'`?

Comment (ClearlyClaire) [reply app/controllers/authorize_interactions_controller.rb:41]: It's a possibility, but we also have classes using `Url`, for some reason I thought it was easier this way.

Comment (ClearlyClaire) [reply app/controllers/authorize_interactions_controller.rb]: Added a commit adding the `URL` inflection and changing class names accordingly.

Comment (nightpool) [reply app/controllers/authorize_interactions_controller.rb]: awesome! I agree that URL is probably the best to standardize on.

**Review by Gargron — APPROVED** [review]

**Pull request merged**